optapy-core/tests/solverConfig-simple.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<solver xmlns="https://www.optaplanner.org/xsd/solver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Is this a minimal solver config that suffices the purpose of the test? If not, let's remove everything the test does not need.